VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts

```diff
@@ -2,6 +2,7 @@ import { ParticipantForUserResponse, Role, RoomSummaryResponse } from 'src/app/s
 import { LinkedParticipantPanelModel } from 'src/app/waiting-space/models/linked-participant-panel-model';
 import { PanelModel } from 'src/app/waiting-space/models/panel-model-base';
 import { ParticipantPanelModel } from 'src/app/waiting-space/models/participant-panel-model';
+import { HearingRole } from 'src/app/waiting-space/models/hearing-role-model';
 
 export class ParticipantPanelModelMapper {
     mapFromParticipantUserResponse(pats: ParticipantForUserResponse[]): PanelModel[] {
```

Comment:
is the "pats" argument not actually used in this function? Or am I reading it worng

Reply:
the **pats** argument is used to return a new var based on the mapping result scenarios